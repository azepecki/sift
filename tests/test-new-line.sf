def int main() { 
    print("Hello world \n this is sift");
    return 0;
}
