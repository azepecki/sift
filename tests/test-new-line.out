Hello world 
 this is sift
